herddb-core/src/test/java/herddb/index/brin/BlockRangeIndexConcurrentTest.java

```diff
@@ -93,16 +93,34 @@ private void dumpIndex(BlockRangeIndex<?, ?> index) {
     private void verifyIndex(BlockRangeIndex<Sized<Integer>, Sized<String>> index) {
         Integer lastmax = null;
         for (Block b : index.getBlocks().values()) {
-            System.out.println("check block " + lastmax + " -> " + ((Sized<Integer>) b.minKey).dummy + "," + ((Sized<Integer>) b.maxKey).dummy);
-            if (lastmax == null) {
-                lastmax = ((Sized<Integer>) b.maxKey).dummy;
+            if (b.key == BlockRangeIndex.BlockStartKey.HEAD_KEY.HEAD_KEY) {
+                System.out.println("check block " + lastmax + " -> -inf," + ((Sized<Integer>) b.maxKey).dummy);
```

Comment:
maybe system.out is slowing down the test and introduces implicit barriers.
can we remove it ?

Reply:
No is on a "verify" test method ran at test end to check the index status. No concurrency nor updates is running at such time